### PR TITLE
Create tests for MentorStudent Model and View

### PR DESCRIPTION
**Pull request by tarassito**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/tests/unittests/mentor/testmodel.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                                                   team=team)
+            curriculum.save()
+
+            topic_python = Topic(id=200,
+                                 curriculum=curriculum,
+                                 title='Python',
+                                 description='My awesome topic')
+            topic_python.save()
+
+            mentor2 = MentorStudent(id=500,
```
Comment (lhalam) [inline eventually/tests/unittests/mentor/testmodel.py]: reuse `mentor` variable

File: eventually/tests/unittests/mentor/testmodel.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                                    mentor=custom_user_first,
+                                    student=custom_user_second,
+                                    topic=topic_python,
+                                    is_done=0)
+            mentor2.save()
+
+            mentor3 = MentorStudent(id=501,
```
Comment (lhalam) [inline eventually/tests/unittests/mentor/testmodel.py]: reuse `mentor` variable

File: eventually/tests/unittests/mentor/testmodel.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                                    mentor=custom_user_first,
+                                    student=custom_user_third,
+                                    topic=topic_python,
+                                    is_done=1)
+            mentor3.save()
+
+            mentor4 = MentorStudent(id=502,
```
Comment (lhalam) [inline eventually/tests/unittests/mentor/testmodel.py]: reuse `mentor` variable

File: eventually/tests/unittests/mentor/testmodel.py [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+    def setUp(self):
+        """Method that provides preparation before testing MentorStudent model's features."""
+        with mock.patch('django.utils.timezone.now') as mock_time:
+            mock_time.return_value = TEST_TIME
+
+            custom_user_first = CustomUser(id=100,
+                                           first_name='petro',
+                                           last_name='hanchuk',
+                                           middle_name='mykola',
+                                           email='email0',
```
Comment (lhalam) [inline eventually/tests/unittests/mentor/testmodel.py]: set valid email

File: eventually/tests/unittests/mentor/testmodel.py [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+                                           password='pas0')
```
Comment (lhalam) [inline eventually/tests/unittests/mentor/testmodel.py]: use `set_password` method

File: eventually/tests/unittests/mentor/testmodel.py [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+                                    mentor=None,
+                                    student=custom_user_third,
+                                    topic=topic_python,
+                                    is_done=1)
+            mentor4.save()
+
+    def test_mentor_to_dict(self):
+        """Method that tests `to_dict` method of certain MentorStudent instance."""
+
+        mentor2 = MentorStudent.objects.get(id=500)
```
Comment (lhalam) [inline eventually/tests/unittests/mentor/testmodel.py]: use `mentor `don't use numbers in variable name

File: eventually/tests/unittests/mentor/testmodel.py [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+        actual_mentor.update(mentor=mentor, student=student, topic=topic, is_done=1)
+        expected_mentor = MentorStudent.objects.get(id=500)
+
+        self.assertEqual(actual_mentor, expected_mentor)
+
+    def test_mentor_get_my_students(self):
+        """Method that tests "get_my_students" method of MentorStudent class object."""
+
+        get_my_students = MentorStudent().get_my_students(100)
+        result = [c for c in get_my_students]
```
Comment (lhalam) [inline eventually/tests/unittests/mentor/testmodel.py]: fix

File: eventually/tests/unittests/mentor/testmodel.py [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+        expected_result = [MentorStudent(id=500), MentorStudent(id=501)]
+        self.assertEqual(result, expected_result)
+
+    def test_mentor_get_available_students(self):
+        """Method that tests "get_available_students" method of MentorStudent class object."""
+
+        get_available_students = MentorStudent().get_available_students()
+        result = [c for c in get_available_students]
+        expected_result = [MentorStudent(id=502)]
+        self.assertEqual(result, expected_result)
```
Comment (lhalam) [inline eventually/tests/unittests/mentor/testmodel.py]: add endline

**Review by VitalyiKhlivnyi — APPROVED** [review]

**Pull request merged**